pandas/core/indexes/base.py

```diff
@@ -1765,6 +1765,29 @@ def _validate_names(
 
         return new_names
 
+    def get_default_index_names(self, names=None, default=None) -> Sequence[str]:
```

Comment:
can you type and add a doc-string

Reply:
Have added doc-string for this function `get_default_index_names`